avxjudge.py

```diff
@@ -107,7 +107,7 @@ def is_avx2(instruction:str, args:str) -> float:
     if "ymm" in args:
         if ("pd" in instruction or "ps" in instruction or instruction in avx2_instructions_ymm) and "xor" not in instruction and "vmov" not in instruction:
             val = 1.0
-        else:
+        if "@" not in args and "<" not in args:
```

Comment:
Why '@'? The '<' I get it, since it's part of a symbol. Can you give an example of an instruction where the '@' would apply without the bracket?

Reply:
They are simple pattern matching for following instructions 

AVX2 instruction ? jmp    39092d <zsymm3m_olcopyi@@Base+0x1f8d>
 AVX2 instruction ? jl     39092a <zsymm3m_olcopyi@@Base+0x1f8a>
 AVX2 instruction ? jne    3909d8 <zsymm3m_olcopyi@@Base+0x2038>
 AVX2 instruction ? jg     390a20 <zsymm3m_olcopyi@@Base+0x2080>
 AVX2 instruction ? jmpq   38ef4e <zsymm3m_olcopyi@@Base+0x5ae>
 AVX2 instruction ? callq  857c0 <ssymm_@plt>
 AVX2 instruction ? je     390aa1 <zsymm3m_olcopyi@@Base+0x2101>

To be honest, I couldn’t figure out an instructions line with ‘@‘ but without ‘<‘ or ‘>’,  so
just make sure such matching is more strict in case there are some lines have ‘@‘ only, make sense ?

Reply:
They are simple pattern matching for following instructions

AVX2 instruction ? jmpq   38eefe <zsymm3m_olcopyi@@Base+0x55e>
AVX2 instruction ? je     390aa1 <zsymm3m_olcopyi@@Base+0x2101>
AVX2 instruction ? jg     38ef87 <zsymm3m_olcopyi@@Base+0x5e7>
AVX2 instruction ? jmpq   38eff6 <zsymm3m_olcopyi@@Base+0x656>
AVX2 instruction ? jmpq   38f264 <zsymm3m_olcopyi@@Base+0x8c4>
AVX2 instruction ? jmpq   39073e <zsymm3m_olcopyi@@Base+0x1d9e>
AVX2 instruction ? jmpq   39042e <zsymm3m_olcopyi@@Base+0x1a8e>
AVX2 instruction ? jmp    3908c3 <zsymm3m_olcopyi@@Base+0x1f23>
AVX2 instruction ? jg     3908c0 <zsymm3m_olcopyi@@Base+0x1f20>
AVX2 instruction ? jmpq   39073e <zsymm3m_olcopyi@@Base+0x1d9e>
AVX2 instruction ? jmp    39092d <zsymm3m_olcopyi@@Base+0x1f8d>
AVX2 instruction ? jl     39092a <zsymm3m_olcopyi@@Base+0x1f8a>
AVX2 instruction ? jne    3909d8 <zsymm3m_olcopyi@@Base+0x2038>
AVX2 instruction ? jg     390a20 <zsymm3m_olcopyi@@Base+0x2080>
AVX2 instruction ? jmpq   38ef4e <zsymm3m_olcopyi@@Base+0x5ae>
AVX2 instruction ? callq  857c0 <ssymm_@plt>
AVX2 instruction ? je     390aa1 <zsymm3m_olcopyi@@Base+0x2101>

To be honest, I couldn’t figure out an instructions line with ‘@‘ but without ‘<‘ or ‘>’, so
just make sure such matching is more strict in case there are some lines have ‘@‘ only, make sense ?

Reply:
They are simple pattern matching for following instructions 

AVX2 instruction ? jmp    39092d <zsymm3m_olcopyi@@Base+0x1f8d>
 AVX2 instruction ? jl     39092a <zsymm3m_olcopyi@@Base+0x1f8a>
 AVX2 instruction ? jne    3909d8 <zsymm3m_olcopyi@@Base+0x2038>
 AVX2 instruction ? jg     390a20 <zsymm3m_olcopyi@@Base+0x2080>
 AVX2 instruction ? jmpq   38ef4e <zsymm3m_olcopyi@@Base+0x5ae>
 AVX2 instruction ? callq  857c0 <ssymm_@plt>
 AVX2 instruction ? je     390aa1 <zsymm3m_olcopyi@@Base+0x2101>
To be honest, I couldn’t figure out a instructions line with ‘@‘ but without ‘<‘ and ‘>’,  so
just make sure such matching is more strict in case there are some lines have ‘@‘ only, make sense ?

Reply:
OOps, 
"AVX2 instruction ? callq 857c0 <ssymm_@plt>" lost '<' and '>' after post.